lib/class-factory.js

```diff
@@ -139,8 +140,27 @@ function ClassFactory (vm) {
     }
   });
 
+  function getUsedClass(className) {
+    let kclass;
+    while ((kclass = classes[className]) === PENDING_USE) {
+      Thread.sleep(0.05);
+    }
+    if (!kclass) {
```

Comment:
Checks should be explicit:

```suggestion
    if (kclass === undefined) {
```